include/af/data.h

```diff
@@ -415,18 +415,10 @@ namespace af
     */
     AFAPI void replace(array &a, const array  &cond, const double &b);
 #endif
-
-    /**
-      @}
-    */
 }
-#endif
```

Comment:
I think this is also necessary. It excludes the af namespace from C headers.

Reply:
No, its not needed, but we can keep them for readability.